x-element.js

```diff
@@ -292,7 +292,7 @@ export default class XElement extends HTMLElement {
     const attributes = new Set();
     const inputMap = new Map();
     for (const [key, property] of entries) {
-      if (!property.internal) {
+      if (!property.internal && XElement.__serializableTypes.has(property.type)) {
```

Comment:
@theengineear can you clarify what is changing here?

Reply:
@theengineear thanks for the clarification in the main thread — I understand the conceptual change but not the line level change. It seems that we're introducing a more explicit check that wasn't previously in place, that's all I'm wondering about.

Reply:
Ah, sorry I misunderstood! This part of attribute validation only applies to properties with attributes. Previously, only internal properties did not have an associated attribute. NOW, because non-serializable properties no longer have attributes, there's no need to check them either.